features/support/capybara.rb

```diff
@@ -97,7 +97,7 @@ def is_a?(klass)
 Capybara.register_driver :headless_chrome do |app|
   options = Selenium::WebDriver::Chrome::Options.new
 
-  options.add_argument('--headless')
+  options.add_argument('--headless=new')
```

Comment:
this seems to be a valid change to merge, although apparently it didn't fix the chrome 127 issue

Reply:
thanks! and yes... didn't fix :sob: